howto/addons/migrate.md

```diff
@@ -1,9 +1,9 @@
-Starting with Anbox Cloud 1.12, the following hooks are deprecated and should be replaced:
+Starting with Anbox Cloud 1.12, use the following new hooks instead of the deprecated ones:
 
-- `install` should be replaced with `pre-start`
-- `prepare` should be replaced with `post-start`
-- `restore` should be replaced with `pre-start`
-- `backup` should be replaced with `post-stop`
+* Use `pre-start` instead of `install`
```

Comment:
hmm, we should drop this line as the `install` hook is addon version free and it doesn't correlate with `pre-start`.
@morphis could you double-check this as well?

Reply:
What do you mean that `install` doesn't correlate with `pre-start`?

Reply:
hmm, if I thought the `install` hook is a special one unlike `restore`, `backup`, `prepare` ones which are bound to a specific addon version 1. But I was wrong obviously and we did deprecate `install` hook for quite a long time.
Everything is clear to me now.
Thanks for the confirmation